docs/source/sensors.rst

```diff
@@ -192,6 +192,90 @@ API Docs
 .. autoclass:: st2reactor.container.sensor_wrapper.SensorService
     :members:
 
+Partitioning Sensors
```

Comment:
It's great to have it documented and docs coming with the feature. 

I would just prefer to somehow separate basic and advanced topics. Else, sensors.rst may become a monster doc and difficult to read through. Think about it (minor).